docs/connecting-data/connecting-to-apis-and-external-data/servicenow-api.md

```diff
@@ -1,7 +1,150 @@
+
+<figure markdown>
+  ![Data Source Wizard][image-7]{ width="100%" }
+</figure>
+
+
+
+On the API screen, select ServiceNow from the list of API forms.
+
+<figure markdown>
+  ![List of APIs][image-8]{ width="100%" }
+</figure>
+
+
+
+The following form will appear. Instructions can be found below on how to find the information required to fill each field on the ServiceNow API form.
+
+Once you have filled out all the fields, press the green 'Save' button, and your API will be connected!
+
+<figure markdown>
+  ![ServiceNow Form][image-1]{ width="100%" }
+</figure>
+
+
+### Name
+Enter any name that will help you recognize this data source from within your query window.
+
+Acceptable characters include:
+
+- lowercase alphanumeric characters
+- underscores
+
+### Instance ID
+
+
+??? example "Steps for finding your API credentials"
+
+    Head over to your ServiceNow [developer portal](https://developer.servicenow.com/dev.do).
+    In the top right corner click the profile icon.
+    
+    ![ServiceNow Home Page][image-10]
+    
+    Click the "Manage instance password" tab.
+    
+    ![Find Instance creds][image-11]
+    
+    Here you will find your credentials necessary to connect your ServiceNow instance to DataDistillr.
+    
+    ![Instance creds][image-12]
+
+
+## Endpoints
+The table below shows a list of endpoints available to connect within the DataDistillr application. If you need to connect to an endpoint not listed below, please use the [Custom API](custom-apis.md) Form.
+
+| Endpoint   | Required | URL Params  | Optional | Description                                      |
+|------------|----------|-------------|----------|--------------------------------------------------|
+| `tables`   |          | `tableName` |          | Returns the table with the matching `tableName`. |
+| `case`     |          |             |          | Returns Customer Service Management (CSM) cases. |
+| `user`     |          |             |          | Returns set of (CSM) accounts.                   |
+| `consumer` |          |             |          | Returns set of (CSM) consumer records.           |
+| `contact`  |          |             |          | Returns set of (CSM) consumer contacts.          |
+
+
+
+### Nav Tree
+The endpoints above will display as follows in the nav tree once your API has successfully connected.
+
+<figure markdown>
+  ![ServiceNow Endpoints][image-2]{ width="100%" }
+</figure>
+
+
+## Sample Queries
+The following queries are intended to help you get started, and make life simpler querying within your API.
+
+For the following examples, suppose that my ServiceNow API data source was called `myservicenowapi` and I want to query an endpoint. The endpoint goes after the ServiceNow data source name like so: ``#!sql FROM `myServiceNowapi`.`<ENDPOINT>` ``.
+
+### Get Tables
```

Comment:
Add a quick explanation to each of the below sections to explain what they show/provide

Reply:
It says that in the endpoints table, should I repeat here again?

Reply:
See how it was done in JIRA - it seems to provide more detail than the table.